### PR TITLE
Update webinar page with new listings

### PR DESCRIPTION
**Pull request by zchase**
This PR adds a bunch of new webinars to the webinar page. The source was the following spreadsheet and a few @stack72 mentioned in Slack: https://docs.google.com/spreadsheets/d/1N-ocz_I6VHJ2f-X3Xq-8UC--DRURbw7M7WHE6nPrtvI/edit#gid=1983058347

cc// @isaac-pulumi

### CODE REVIEW

File: content/webinars/ndc-oslo-2020-06-12/index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# Gated webinars will have a registration form and the user will need
+# to fill out the form before viewing.
+gated: false
+
+# The layout of the landing page.
+type: webinars
+
+# External webinars will link to an external page instead of a webinar
+# landing/registration page.
+external: false
```
Comment (isaac-pulumi) [inline content/webinars/ndc-oslo-2020-06-12/index.md:30]:
```suggestion
external: true
```

File: content/webinars/ndc-oslo-2020-06-12/index.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+# The url slug for the webinar landing page. If this is an external
+# webinar, use the external URL as the value here.
+url_slug: "https://ndcoslo.com/talk/infrastructure-as-software/"
```
Comment (isaac-pulumi) [inline content/webinars/ndc-oslo-2020-06-12/index.md:34]: Link didn't work -- appears to need an 'external' tag?

File: content/webinars/dev-095-2020-06-11/index.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+# Gated webinars will have a registration form and the user will need
+# to fill out the form before viewing.
+gated: false
+
+# The layout of the landing page.
+type: webinars
+
+# External webinars will link to an external page instead of a webinar
+# landing/registration page.
+external: false
```
Comment (isaac-pulumi) [inline content/webinars/dev-095-2020-06-11/index.md:30]:
```suggestion
external: true
```

File: content/webinars/dev-095-2020-06-11/index.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+# The url slug for the webinar landing page. If this is an external
+# webinar, use the external URL as the value here.
+url_slug: "https://www.meetup.com/nycdevops/events/270997684/"
```
Comment (isaac-pulumi) [inline content/webinars/dev-095-2020-06-11/index.md:34]: This link is also broken. Needs to be marked external?

File: content/webinars/spinnaker-live-2020-06-18/index.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
+---
+# Name of the webinar.
+title: "Spinnaker Live"
+meta_desc: "In this talk, Dan and Praneet will go over how the Pulumi plugin for spinnaker works and example use cases."
```
Comment (isaac-pulumi) [inline content/webinars/spinnaker-live-2020-06-18/index.md:4]:
```suggestion
meta_desc: "Dan and Praneet will demonstrate how Pulumi works with Spinnaker to enable cloud-native continuous delivery for infrastructure along with your applications."
```

**Comment by zchase** [comment]
@justinvp I fixed the grammar issue, but for last names, we are going to leave them out for now. Adding the last name puts us over the Meta Description character limit in some cases. To avoid having to rework the descriptions to make the last name fit we will just keep it as is.

**Comment by cnunciato** [comment]
> but for last names, we are going to leave them out for now

Let's please see if we can figure out how to do this without just cutting out part of people's names.

File: content/webinars/stack-conf-eu-2020-06-16/index.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
+---
+# Name of the webinar.
+title: "StackConf Online"
+meta_desc: "In this talk, Paul will demonstrate how to write infrastructure code in TypeScript, Python, DotNet & Go across multiple clouds."
```
Comment (cnunciato) [inline content/webinars/stack-conf-eu-2020-06-16/index.md:4]: It's `.NET`, not `Dotnet`.

Comment (justinvp) [reply content/webinars/stack-conf-eu-2020-06-16/index.md:4]: 👍 great catch!

File: content/webinars/ndc-oslo-2020-06-12/index.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
+---
+# Name of the webinar.
+title: "NDC Oslo"
+meta_desc: "In this talk, Paul will demonstrate why writing infrastructure in general programming languages is a better choice for infrastructure management."
```
[the remark this answers is not in the document]
Comment (zchase) [reply content/webinars/ndc-oslo-2020-06-12/index.md:4]: 👍

File: content/webinars/spinnaker-live-2020-06-18/index.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
+---
+# Name of the webinar.
+title: "Spinnaker Live"
+meta_desc: "Dan and Praneet will demonstrate how Pulumi works with Spinnaker to enable cloud-native continuous delivery for infrastructure along with your applications."
```
[the remark this answers is not in the document]
Comment (zchase) [reply content/webinars/spinnaker-live-2020-06-18/index.md:4]: 👍

File: content/webinars/fsharp-conf-2020-06-05/index.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
+---
+# Name of the webinar.
+title: "fsharpConf 2020"
+meta_desc: "In this talk, Mikhail will introduce Pulumi, an open-source tool that leverages programming languages to manage infrastructure, yet in a declarative manner."
```
[the remark this answers is not in the document]
Comment (zchase) [reply content/webinars/fsharp-conf-2020-06-05/index.md:4]: 👍

File: content/webinars/stack-conf-eu-2020-06-16/index.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (zchase) [reply content/webinars/stack-conf-eu-2020-06-16/index.md:4]: 👍

File: content/webinars/tech-strong-con-2020-06-04/index.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
+---
+# Name of the webinar.
+title: "TechStrong Con"
+meta_desc: "In this talk, Paul will demonstrate how to build infrastructure that manages Serverless, Kubernetes, PaaS and IaaS systems across multiple cloud providers."
```
[the remark this answers is not in the document]
Comment (zchase) [reply content/webinars/tech-strong-con-2020-06-04/index.md:4]: 👍

File: content/webinars/stack-conf-eu-2020-06-16/index.md (hunk 6 above)
Comment (zchase) [reply content/webinars/stack-conf-eu-2020-06-16/index.md:4]: 👍

**Comment by cnunciato** [comment]
ndc-oslo.png and dev095.png are both >= 1.5 MB. Can that be better?

We should probably run all images through some sort of an optimization process and watermarks them or something, so we aren't having to look at these individually on every PR.

File: content/webinars/dev-095-2020-06-11/index.md [hunk 11]
```diff
@@ -0,0 +1,66 @@
+---
+# Name of the webinar.
+title: "DevO95 - A Day of DevOps Meetups"
+meta_desc: "In this talk, learn about the next evolution of Infrastructure as Code, Pulumi and how it makes working with the cloud easier using real programming languages."
```
[the remark this answers is not in the document]
Comment (zchase) [reply content/webinars/dev-095-2020-06-11/index.md:4]: @cnunciato Certainly open to suggestions here but yes this was lifted from the event description.

[the remark this answers is not in the document]
Comment (justinvp) [reply content/webinars/dev-095-2020-06-11/index.md:4]: Pretty generic, but maybe something like:

```suggestion
meta_desc: "Learn how Infrastructure as Code with Pulumi makes working with the cloud easier using real programming languages."
```

?

**Pull request merged**